src/pages/workspaces/workspace/Tools.js

```diff
@@ -269,15 +290,24 @@ export const Tools = _.flow(
     return _.find({ namespace, name }, configs)
   }
 
+  addRedactedAttribute(config, methods) {
+    const { methodName, methodNamespace, methodVersion, sourceRepo } = config.methodRepoMethod
+    const isRedacted = _.some({ name: methodName, namespace: methodNamespace, snapshotId: methodVersion }, methods)
+    if (!sourceRepo) reportError('Caller must specify source repo for method')
```

Comment:
Do we need to check for this here? Could this even happen in the normal course of things?

Reply:
They have the check in FC, so I think there may be a case for it? It could also be should it somehow happen, we still handle it. 

That being said, the ternary would just return false if we were to remove check and there wasn't a source repo. What do you think?

Reply:
Resolved: remove check since we handle it if there isn't a source repo by returning false